app/src/main/res/values/strings.xml

```diff
@@ -896,6 +896,8 @@
     GND. You can use any of the PSLab device GND pins.\n</string>
 
     <string name="barometer">Barometer</string>
+    <string name="barometer_alert_title">Sensor Not Present</string>
+    <string name="barometer_alert_description">Barometer sensor is not present in your device, you need to use external sensor(BMP180) with PSLab for barometer</string>
```

Comment:
This line isn't proper. You could rather use
"Barometer sensor is not present in your device. Please connect an external sensor(BMP180) with the PSLab I2C pin for using Barometer instrument"